tests/test_simple.py

```diff
@@ -702,6 +702,53 @@ def test_version_management(self):
         self.assertTrue(b > c)
         self.assertTrue(a > c)
 
+    def test_pids(self):
```

Comment:
Maybe we could write a helper function to check if `psutil` is available and then use it in `@unittest.skipUnless()`.

Reply:
In this case it's tested for the case when pytest is unavailable too, so it should not be skipped.